scripts/ccpp_suite.py

```diff
@@ -741,22 +739,22 @@ def write_suite_part_list_sub(self, ofile, errmsg_name, errcode_name):
         self._errcode_var.write_def(ofile, 2, self)
         else_str = ''
         ename = self._errcode_var.get_prop_value('local_name')
-        ofile.write("{} = 0".format(ename), 2)
+        ofile.write(f"{ename} = 0", 2)
         ename = self._errmsg_var.get_prop_value('local_name')
-        ofile.write("{} = ''".format(ename), 2)
+        ofile.write(f"{ename} = ''", 2)
         for suite in self.suites:
             oline = "{}if(trim(suite_name) == '{}') then"
             ofile.write(oline.format(else_str, suite.name), 2)
             API.write_var_set_loop(ofile, 'part_list', suite.part_list(), 3)
             else_str = 'else '
         # end for
         ofile.write("else", 2)
-        emsg = "write({errmsg}, '(3a)')".format(errmsg=errmsg_name)
+        emsg = f"write({errmsg_name}, '(3a)')"
         emsg += "'No suite named ', trim(suite_name), ' found'"
         ofile.write(emsg, 3)
-        ofile.write("{errcode} = 1".format(errcode=errcode_name), 3)
+        ofile.write(f"{errcode_name} = 1", 3)
         ofile.write("end if", 2)
-        ofile.write("end subroutine {}".format(API.__part_fname), 1)
+        ofile.write(f"end subroutine {API.__part_fname}", 1)
```

Comment:
Has someone gone through the entire code and replaced old-style `.format`-strings with f-strings? If not, that might be a good cleanup task.

Reply:
There's now a separate issue, for this, so marking as resolved